libs/metrics/social_distancing.py

```diff
@@ -10,11 +10,12 @@
 from libs.utils.loggers import get_source_log_directory
 
 from .base import BaseMetric
+from constants import SOCIAL_DISTANCING
 
 
 class SocialDistancingMetric(BaseMetric):
 
-    reports_folder = "social-distancing"
+    reports_folder = SOCIAL_DISTANCING
```

Comment:
Should we also use the constants in the facemask metrics?

Reply:
The folder for facemask is `face-mask-usage`, where as the c constant is `facemask-usage`. Chaning one would break some cases data is already stored there. We could maybe internally use `face-mask-usage` and change the constant but maybe it's best doing so in a separate PR